pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -291,7 +304,9 @@ def test_sc_count_sql_table():
 def test_get_sql_table_col_names():
     # Iterate over initialized compartments
     for compartment in AP.compartments:
-        meta_cols, feat_cols = AP.get_sql_table_col_names(table=compartment)
+        meta_cols, feat_cols = AP.split_columns_into_classes(
```

Comment:
rather than chaining these two function calls together, can you explicitly test both of them separately?

This will enable us to zero in on code changes that might break one but not the other.

e.g.

```python
col_name_result = AP.get_sql_table_col_names(table=compartment)
assert col_name_result == expected_result

meta_cols, feat_cols = AP.split_columns_into_classes(col_name_result)
# Then pass these vars into existing assertions
```

Reply:
Done!